velox/docs/functions/spark/array.rst

```diff
@@ -198,3 +198,19 @@ Array Functions
 
         SELECT transform(array(1, 2, 3), x -> x + 1); -- [2,3,4]
         SELECT transform(array(1, 2, 3), (x, i) -> x + i); -- [1,3,5]
+
+.. spark:function:: array_insert(array(E), pos, E, legacyNegativeIndex) -> array(E)
+
+    Places new element into index pos of the input array.
+    Returns NULL if the input array or index is NULL. Array indices start at 1. The maximum negative index
+    is -1. When legacyNegativeIndex is true, the function is 0-based for negative indexes which means -1
```

Comment:
> The maximum negative index is -1.

Could we explain more about its meaning, and does it vary according to `legacyNegativeIndex`?

Reply:
Updated